File: py_coding_q_and_a/08.py
# // Print the Fibonacci sequence up to n terms.

def fibo(n):
    res=[0,1]
    first,second=res

    while n >=2:
        newNum=first+second
        first=second
        second=newNum
        res.append(newNum)
        n=n-1
    return res

print(fibo(5))

File: js_coding_q_and_a/08.js
// Print the Fibonacci sequence up to n terms.

const fibo = (n) => {
  var first = 0;
  var second = 1;
  var res = [first, second];
  while (n >= 2) {
    var newNum = first + second;
    first = second;
    second = newNum;
    res.push(newNum);
    n = n - 1;
  }
  return res;
};

console.log(fibo(5));
